tasks/config-map/task.json

```diff
@@ -12,27 +12,81 @@
   ],
   "preview": true,
   "demands": [],
+  "groups": [
+    {
+      "name": "openshiftCluster",
+      "displayName": "OpenShift Cluster",
+      "isExpanded": true
+    },
+    {
+      "name": "commands",
+      "displayName": "Commands",
+      "isExpanded": true
+    }
+  ],
   "version": {
     "Major": 2,
     "Minor": 0,
-    "Patch": 12
+    "Patch": 13
   },
   "minimumAgentVersion": "1.95.0",
   "instanceNameFormat": "config-map $(message)",
   "inputs": [
+    {
+      "name": "connectionType",
+      "type": "pickList",
+      "label": "Service connection type",
+      "defaultValue": "OpenShift Connection Service",
+      "required": true,
+      "options": {
+        "OpenShift Connection Service": "OpenShift Connection Service",
+        "File Configuration": "File Configuration",
```

Comment:
I would follow the way kubectl works. I'd suggest substitute them with a new entry (e.g "Runtime Configuration": "Set Up Configuration on Runtime") and then when selected, show the radio type with the two options you already created (file config, inline config.).